loom/script/reflection/lsMethodInfo.cpp

```diff
@@ -69,7 +75,8 @@ void MethodBase::push()
 {
     if (!attr.isNative)
     {
-        assert(byteCode);
+        // Why was this here?
```

Comment:
This is to ensure that if we have a non-native method we have bytecode for it... is that an invalid assumption?